tests/modules/programs/gnome-terminal/gnome-terminal-1.nix

```diff
@@ -0,0 +1,56 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.gnome-terminal = {
+      enable = true;
+      profile = {
+        "e0b782ed-6aca-44eb-8c75-62b3706b6220" = {
+          allowBold = true;
+          audibleBell = true;
+          backspaceBinding = "ascii-delete";
+          boldIsBright = true;
+          colors = {
+            backgroundColor = "#2E3436";
+            foregroundColor = "#D3D7C1";
+            palette = [
+              "#000000"
+              "#AA0000"
+              "#00AA00"
+              "#AA5500"
+              "#0000AA"
+              "#AA00AA"
+              "#00AAAA"
+              "#AAAAAA"
+              "#555555"
+              "#FF5555"
+              "#55FF55"
+              "#FFFF55"
+              "#5555FF"
+              "#FF55FF"
+              "#55FFFF"
+              "#FFFFFF"
+            ];
+          };
+          cursorBlinkMode = "off";
+          cursorShape = "underline";
+          default = true;
+          deleteBinding = "delete-sequence";
+          scrollbackLines = 1000000;
+          scrollOnOutput = false;
+          showScrollbar = false;
+          transparencyPercent = 5;
+          visibleName = "kamadorueda";
+        };
+      };
+      showMenubar = false;
+    };
+
```

Comment:
Oh, I think there will have to be an overlay here that replaces `gnome3.gnome-terminal` by a dummy package.

Reply:
May I know why? is it for performance reasons?

Reply:
Yes, it is unnecessary to download the entire Gnome Terminal closure when its not actually used. You can see examples in the other tests by searching for "dummy".